pynest/examples/brunel_alpha_evolution_strategies.py

```diff
@@ -64,6 +64,7 @@
 
 '''
 
+from __future__ import print_function
```

Comment:
Because of the `end=' '` here:
https://github.com/nest/nest-simulator/blob/4b9227814595259da12653dd65745d1f8514b16a/pynest/examples/brunel_alpha_evolution_strategies.py#L558

Reply:
OK.